src/components/SettingsTextSize.tsx

```diff
@@ -55,7 +55,7 @@ const SettingsTextSize = ({ userSettings, setUserSettings }: Props) => {
           <Label>Change Text Size</Label>
           <FontSizeButtons>
             <SegmentedButton data-testid="change-text-size-buttons">
-              {FONT_SIZES.map((v: number, i: number) => (
+              {FONT_SIZES.slice(0, 3).map((v: number, i: number) => (
```

Comment:
There are four font sizes, but we can only access three of them?

Reply:
Yes. This is to allow the VxPrint UI to use the same `SettingsTextSize` mechanism to display larger content. Eventually we should replace this simple changing of the root `font-size` with "themes".